src/databricks/labs/ucx/cli.py

```diff
@@ -146,6 +147,26 @@ def repair_run(w: WorkspaceClient, step):
     installer.repair_run(step)
 
 
+@ucx.command
+def validate_groups_permissions(w: WorkspaceClient):
+    """Validate that all the crawled permissions are applied correctly to the destination groups"""
+    logger.info("Running validation of permissions applied to groups.")
+    installation_manager = InstallationManager(w)
+    installation = installation_manager.for_user(w.current_user.me())
```

Comment:
```suggestion
    installation = Installation.current(w, "ucx")
```

Reply:
we use the same in all cli commands. Installation does not have "current" method

Reply:
https://github.com/databrickslabs/ucx/pull/860 there's no more installation manager

Reply:
got it but the other PR must be merged first

Reply:
merged :)